.circleci/config.yml

```diff
@@ -65,13 +65,16 @@ jobs:
           name: Install Vue3
           command: |
             yarn remove vue vue-jest vue-template-compiler babel-preset-es2015
-            yarn add vue@3.0.6 vue-jest@5.0.0-alpha.10 vue-loader@16.1.2 jest@26.6.3 babel-jest@25.2.6 @babel/preset-env@7.14.5 -D -E
-            echo "export * from './index-3';" > src/util/vue/index.js
+            yarn add vue@3.1.2 @vue/server-renderer@3.1.2 vue-jest@5.0.0-alpha.10 vue-loader@16.1.2 jest@26.6.3 babel-jest@25.2.6 @babel/preset-env@7.14.5 @babel/plugin-transform-runtime@7.14.5 -D -E
```

Comment:
this is starting to be a lot of code, shouldn't it be in a dedicated script instead of inline in yaml?

Reply:
I'll do it in a separate PR (because there are other stacked PRs).